jam/server/api/jsonapi.py

```diff
@@ -0,0 +1,131 @@
+import functools
+import http.client
+import json
+import operator
+import re
+
+import tornado.web
+
+import jam
+from jam.server.api.base import CORSMixin
+
+
+def parse_int(raw, lo=None, hi=None):
```

Comment:
I might rename this to ... something better. enforce_ranged_int, I don't know. It seems like it does more than parse integers.